lib/api.ts

```diff
@@ -180,6 +181,20 @@ export async function getContributorBySlug(file: string, detail = false) {
 
   const calendarData = getCalendarData(weightedActivity.activity);
 
+  const isNewContributor = (() => {
+    const { activity } = activityData;
+
+    const aWeekAgo = sub(new Date(), { days: 7 });
+
+    const firstActivity = activity
+      .filter((act) => act.type === "pr_merged")
+      .sort((a, b) => compareAsc(a.time, b.time))[0];
+
+    if (!firstActivity) return false;
+
+    return isAfter(new Date(firstActivity.time), aWeekAgo);
```

Comment:
i think this is the third time

```suggestion
    return isAfter(firstActivity.time, aWeekAgo);
```

Reply:
Oh sorry forgot about it 😬